tests/test.py

```diff
@@ -807,11 +807,26 @@ def tearDownModule():
     kill_certificate_validation_services()
 
 
-ConnectionArgs = collections.namedtuple("ConnectionArgs",
-    field_names=["user", "password", "host", "port", "dbname",
-                 "ssl_ca", "ssl_key", "ssl_cert", "format"],
-    # 'format' is optional, other fields are required.
-    defaults=[None])
+if sys.version_info[1] > 6:
```

Comment:
So modern Python `>=3.7` requires this new way of creating database connection args, but to make sure it works on 3.6 as well you added this `class` thing?

Reply:
this type is our and not related to python or DBs actually. Namedtuple same as class, but easier to declare. But <=3.6 doesn't support default values. So we use more verbose type description than namedtuple